src/app/ideas/IdeasFeed.js

```diff
@@ -1,24 +1,58 @@
 import React from 'react';
 import styled from 'styled-components';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
 import IdeaPost from './IdeaPost';
 
+library.add(faTimes);
+
 const StyledSection = styled.div`
-  display: grid;
   padding: 10px;
   overflow: scroll;
 `;
 
-export default function IdeasFeed({ posts }) {
+const FilterTitle = styled.div`
+  color: rebeccapurple;
+`;
+
+const StyledTag = styled.span`
+  padding: 3px 6px;
+  margin: 0px 5px 0px 0px;
```

Comment:
i am not sure about the default styling of a span, but maybe you can write `margin-bottom: 5px;`

Reply:
it was `margin-right`, but yeah. I read that convention is putting all the information in the general key, but it is more readable to see that I specifically wanted to _JUST_ pass a `margin-right` 🤷‍♂️